tests/security_features_test.go

```diff
@@ -188,4 +188,39 @@ var _ = Describe("SecurityFeatures", func() {
 			})
 		})
 	})
+
+	Context("Check virt-launcher capabilities", func() {
+		var container k8sv1.Container
+		var vmi *v1.VirtualMachineInstance
+
+		It("has precisely the documented extra capabilities relative to a regular user pod", func() {
+			vmi = tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+
+			By("Starting a New VMI")
+			vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+			Expect(err).ToNot(HaveOccurred())
+			tests.WaitForSuccessfulVMIStart(vmi)
+
+			By("Ensuring VMI is running by logging in")
+			tests.WaitUntilVMIReady(vmi, tests.LoggedInAlpineExpecter)
+
+			By("Fetching virt-launcher Pod")
+			pod := tests.GetPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+			for _, containerSpec := range pod.Spec.Containers {
+				if containerSpec.Name == "compute" {
+					container = containerSpec
+					break
+				}
+			}
+			caps := *container.SecurityContext.Capabilities
+			Expect(len(caps.Add)).To(Equal(3))
+
+			By("Checking virt-launcher Pod's compute container has precisely the documented extra capabilities")
+			for _, cap := range caps.Add {
```

Comment:
You can use ContainElements matcher for this.

Reply:
Thanks for the review comments, I am new to golang/gomega.

I looked at making use of this and I realized we need gomega version `v1.9.0` and I think we are using `v1.7.0`. Maybe we need to update gomega to make use of `ContainElements`.

Reply:
I see... I don't have a strong opinion about this. Yes, we are using gomega `1.7.0`. I think we can update it.